CHANGELOG.md

```diff
@@ -58,6 +58,8 @@ Format:
 - Docassemble Project name prefix now includes ALKiln in it for clarity
 
 ### Fixed
+- DOCKER UPDATE (for ALKilnInThePlayground): Allows ALKilnInThePlayground to upload files (though S3 is not consistent yet). See https://github.com/SuffolkLITLab/docassemble-ALKilnInThePlayground/issues/18. This may require some developers to update their system (updating their docker image) as described in [the documentation](https://suffolklitlab.org/legal-tech-class/docs/practical-guide-docassemble/maintaining-docassemble#updates-to-the-docassemble-container). Updating to a new docker images is something that all docassemble users will have to do sometime anyway, sometimes for security.
```

Comment:
```suggestion
- DOCKER UPDATE (for ALKilnInThePlayground): Allows ALKilnInThePlayground to upload files (though S3 is not consistent yet). See https://github.com/SuffolkLITLab/docassemble-ALKilnInThePlayground/issues/18. This may require some developers to update their system (updating their docker image) if you are below [docassemble-os 1.0.8](https://github.com/jhpyle/docassemble-os/releases/tag/v1.0.8). You are below docassemble-os 1.0.8 if you run `docker image inspect jhpyle/docassemble | grep Created` on your host server (Lightsail, etc.) and the created date is before 2023-04-09. You can update as described in [the documentation](https://suffolklitlab.org/legal-tech-class/docs/practical-guide-docassemble/maintaining-docassemble#updates-to-the-docassemble-container). Updating to a new docker images is something that all docassemble users will have to do sometime anyway, sometimes for security.
```